third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -952,7 +964,7 @@ def to_markdown(
                 These parameters will be passed to `tabulate <https://pypi.org/project/tabulate>`_.
 
         Returns:
-            DataFrame: DataFrame in Markdown-friendly format.
+            bigframes.pandas.DataFrame: DataFrame in Markdown-friendly format.
```

Comment:
I don't think this actually returns a DataFrame object. Pretty sure it returns a string like https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_markdown.html does.

```suggestion
            str: DataFrame in Markdown-friendly format.
```